recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,4 @@
 dnnl_cpu_runtime:
   - tbb  # [not (ppc64le or aarch64)]
   - omp  # [unix]
```

Comment:
If you want to add OpenMP for Windows, you need to remove the `unix` here and rerender.
```suggestion
  - omp
```

Reply:
Fixed